index.html

```diff
@@ -762,9 +762,311 @@
         <!-- ************ -->
 
         <section id="class-3">
-          <h2>Third Class: Jeff</h2>
+          <h2>Third Class</h2>
+          <p>Jeff O'Leary</p>
         </section>
+        <section>
+          <h4>Agenda</h4>
+          <p class="fragment">Navigation</p>
```

Comment:
I think just putting what you're teaching in the context of class vs. instance, properties vs. functions, and subclasses should be fine. In the last class, probably because I didn't get around to showing them around a real app, they had trouble connecting UIViewControllers with those concepts: "Wait, what are we subclassing? Does that mean we have to access UIViewController before we get the `view` property?" "So, we should make the name of this class lowercase?" "Is this an instance? Is that? Is this other thing?"

Also, might make sense to explain that you often use other people's work, you don't have to write everything yourself. In the first class I think I gave that impression, so some of the absolute beginners were confused. They also didn't know how to find the .h files of the classes we were using.